live-examples/js-examples/classes-extends.html

```diff
@@ -0,0 +1,16 @@
+<pre>
+<code id="static-js">class formatDate extends Date {
+  constructor(dateStr) {
+    super(dateStr);
+  }
+
+  getFormattedDate() {
+    var months = ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec'];
```

Comment:
Very nice example, but this line is too long, we should split it (I know this will make the example even longer, but I'd prefer that).

```
    var months = ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun',
                  'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec'];
```

Reply:
Sure. Makes sense!